Dockerfile

```diff
@@ -3,8 +3,8 @@ FROM openjdk:8-jre-slim
 LABEL maintainer="cgiraldo@gradiant.org" \
       organization="gradiant.org"
 
-ARG version=2.7.7
-ENV HADOOP_VERSION=$version \
+ARG HADOOP_VERSION=3.1.3
```

Comment:
Do not change build args, since we use lowercase by agreement in Gradiant. "version" is a common build arg for multiple Dockerfiles at Gradiant.